static/app/views/settings/components/settingsBreadcrumb/types.tsx

```diff
@@ -1,7 +1,11 @@
-import type {RouteProps} from 'sentry/types/legacyReactRouter';
+import type {RouteComponentProps, RouteProps} from 'sentry/types/legacyReactRouter';
 
 // TODO(ts): The `name` attribute doesn't appear on any of the react router route types
 
 export type RouteWithName = RouteProps & {
   name?: string;
 };
+
+export type SettingsBreadcrumbProps = Pick<RouteComponentProps, 'route' | 'routes'> & {
+  isLast: boolean;
```

Comment:
this was totally missing in the types, but it’s drilled down to the underlying `Menu` so we need it.